imbi/automations/models.py

```diff
@@ -46,6 +46,8 @@ def verify_legal_callable(v):
 
 class AutomationCategory(enum.Enum):
     CREATE_PROJECT = 'create-project'
+    CREATE_DEPENDENCY = 'create-project-dependency'
+    REMOVE_DEPENDENCY = 'remove-project-dependency'
```

Comment:
Also need to add these to [v1.automation_category_type in the DB](https://github.com/AWeber-Imbi/imbi-schema/blob/main/types/v1/automation_category_type.sql)

Reply:
bah, forgot to push up the PR